AppCreationScripts/Configure.ps1

```diff
@@ -213,7 +213,7 @@ Function ConfigureApplications
    # Add Required Resources Access (from 'service' to 'Microsoft Graph')
    Write-Host "Getting access from 'service' to 'Microsoft Graph'"
    $requiredPermissions = GetRequiredPermissions -applicationDisplayName "Microsoft Graph" `
-                                                -requiredDelegatedPermissions "openid|profile|offline_access|Mail.Read" `
+                                                -requiredDelegatedPermissions "openid|profile|offline_access|Mail.Read|User.Read" `
```

Comment:
Why do you need User.Read?

Reply:
ViewProfile (msal.js) uses user.read to display the user profile. I modified the app creation scripts to include this but will need to modify manual app creation scripts before merge